concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme.md

```diff
@@ -12,14 +12,30 @@ Configure the module bundler you are using and import `dx.common.css` and a them
 2. Open the DevExtremeBundleConfig.cs file
 3. Find the following code and replace `dx.light.css` in it with the theme stylesheet of your choice:
```

Comment:
Change to:
3. Find the following code and replace `dx.light.css` with the desired theme stylesheet: